extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java

```diff
@@ -88,7 +113,7 @@ public String getId() throws ParsingException {
 
             return element.attr("data-channel-external-id");
         } catch (Exception e) {
-            throw new ParsingException("Could not get channel id", e);
+            return super.getId();
```

Comment:
Are you sure this is ok? You should probably check that `super.getId()` (that is equivalent to `linkHandler.getId()`, see [Extractor.java, line 74](https://github.com/TeamNewPipe/NewPipeExtractor/blob/f250db521e490c6b45059463ac26e4137aaccb44/extractor/src/main/java/org/schabi/newpipe/extractor/Extractor.java#L74) and [LinkHandler.java, line 29](https://github.com/TeamNewPipe/NewPipeExtractor/blob/f250db521e490c6b45059463ac26e4137aaccb44/extractor/src/main/java/org/schabi/newpipe/extractor/linkhandler/LinkHandler.java#L29)) really returns something. If it returns nothing (i.e. `null` or an empty string), just throw to the exception (`throw new ParsingException("Could not get channel id", e);`).